org.b3log.siyuan.yml

```diff
@@ -0,0 +1,67 @@
+app-id: org.b3log.siyuan
+runtime: org.freedesktop.Platform
+runtime-version: &runtime-version '23.08' 
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: *runtime-version
+command: start-siyuan
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --talk-name=org.kde.StatusNotifierWatcher
+modules:
+  - name: siyuan
+    buildsystem: simple
+    build-commands:
+      - mv "siyuan" ${FLATPAK_DEST}/siyuan
+      - rm ${FLATPAK_DEST}/siyuan/chrome-sandbox
+
+      # To allow separate locales
+      # https://searchfox.org/mozilla-central/rev/8a4f55bc09ffc5c25dcb4586c51ae4a9fee77b4c/taskcluster/docker/firefox-flatpak/runme.sh#131-133
+      - |
+        for lang in ${FLATPAK_DEST}/siyuan/locales/*.pak
+        do
+          locale="$(basename -s .pak $lang)"
+          install -Dm644 -t "${FLATPAK_DEST}/share/runtime/locale/${locale%%-*}/" "$lang"
+          ln -sf "${FLATPAK_DEST}/share/runtime/locale/${locale%%-*}/$(basename $lang)" "${FLATPAK_DEST}/siyuan/locales/$(basename $lang)"
+        done
+
+      - install -Dm644 "${FLATPAK_DEST}/siyuan/resources/stage/icon-large.png" "${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png"
+      - install -Dm644 org.b3log.siyuan.desktop "${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop"
+      - install -Dm644 org.b3log.siyuan.metainfo.xml /app/share/metainfo/$FLATPAK_ID.metainfo.xml
+    sources:
+      - type: archive
+        dest: siyuan
+        only-arches: [x86_64]
+        url: https://github.com/siyuan-note/siyuan/releases/download/v2.10.14/siyuan-2.10.14-linux.tar.gz
```

Comment:
so how does this work on the aarch64 flatpak that is being built?

can't it just be built from source instead?